### ISSUE TITLE
Training Datasets Are Faulty

### ISSUE THREAD

**Issue by psb-cc** [AI4Bharat/vistaar#12]
Some of the datasets in the Vistaar-train mentioned in the [Download Training Datasets and Benchmarks](https://github.com/AI4Bharat/vistaar?tab=readme-ov-file#download-training-datasets-and-benchmarks) table in the README file seem to be either lacking transcriptions / are test cuts of the respective datasets. Please let me know if I'm looking in the wrong place.

Here are the issues I'm facing after extracting the `Training Datasets` zip files mentioned in the aforementioned table:

1. commonvoice (https://indicwhisper.objectstore.e2enetworks.net/vistaar/commonvoice.zip): contains only a test manifest file.
![Image](https://github.com/user-attachments/assets/6ab5f844-b6a0-41db-8aa0-f6c364695bd1)

1. fleurs (https://indicwhisper.objectstore.e2enetworks.net/vistaar/fleurs.zip): no issues

1. indictts (https://indicwhisper.objectstore.e2enetworks.net/vistaar/indictts.zip): no issues

1. kathbath (https://indicwhisper.objectstore.e2enetworks.net/vistaar/kathbath.zip): this zip contains test and test-known cuts. Having worked with IndicSUPERB, I know these are test sets and not the training data.
![Image](https://github.com/user-attachments/assets/c09234d5-d5ae-44d6-810b-704a759c4747)

1. kathbath-hard ([kathbath_noisy.zip](https://indicwhisper.objectstore.e2enetworks.net/vistaar/kathbath_noisy.zip)): same issue as kathbath

1. graamvani: this zip also contains the test set and not the training data.
![Image](https://github.com/user-attachments/assets/6da099b4-f5a0-4bcb-9439-fde497866497)

Again, let me know if I'm looking at the wrong place / downloading the wrong zip files.
